tests/resolvers/Post/comments.spec.ts

```diff
@@ -28,9 +39,28 @@ afterAll(async () => {
 
 describe("resolvers -> Post -> comments", () => {
   it(`returns the comment object for parent post`, async () => {
-    const parent = testPost!.toObject();
+    const parent = await Post.findById(testPost?._id);
+
+    const commentsPayload = await commentsResolver?.(
+      parent!.toObject(),
+      {},
+      {}
+    );
+
+    const comments = await Comment.find({
+      postId: testPost!._id,
+    }).lean();
+
+    expect(commentsPayload).toEqual(comments);
+  });
+  it(`returns the comment object for parent post from cache`, async () => {
```

Comment:
Both test cases seem similar how do you assert that the data is coming from cache?

Reply:
Well, the assertion is a little naive but basically, we can say that the data is coming from cache from the code coverage of the file in the tests. 

If the returned data was from the db query (instead of the cahce) then that would mean the conditional statement to check whether the comment exists in the cache -> if true returning the given comment from cache would not be hit and that would have reflected in the code coverage.